kolibri/auth/constants/facility_configuration_presets.json

```diff
@@ -12,6 +12,7 @@
   },
   "nonformal": {
     "name": "Self-managed",
+    "default": true,
```

Comment:
Is this basically the main change? The rest of the diff looks like re-formatting the HTML.

Reply:
Mainly, yeah. The other issue was that the HTML was assigning the wrong radio value to each option